packages/oth/oth.2.10/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "orbitz@gmail.com"
+build: [
+	[make "-j%{jobs}%"]
+]
+
+build-test: [
+	[make "-j%{jobs}%" "test"]
+]
+
+install: [
+	[make "PREFIX=%{prefix}%" "install"]
+]
+
+remove: [
+	[make "PREFIX=%{prefix}%" "remove"]
+]
+
+depends: [
+	"containers"
+	"duration"
+	"merlin-of-pds"
+	"ocamlfind"
+	"pds" { build & (>= "5" & < "6") }
+	"revops"
+]
+
+authors: [
+	"dklee@dklee.org"
+	"orbitz@gmail.com"
+]
+
+homepage: "https://bitbucket.org/mimirops/oth"
+bug-reports: "https://bitbucket.org/mimirops/oth/issues"
+dev-repo: "git@bitbucket.org:mimirops/oth.git"
+
```

Comment:
should be good to merge once we have an `available: [ocaml-version >= "4.03.0"]` since it depends on `type result` existing.